static/src/js/components/CustomRadioWidget/CustomRadioWidget.jsx

```diff
@@ -100,7 +99,7 @@ const CustomRadioWidget = ({
           <input
             className="d-flex form-check-input m-2"
             id={`${id}-true`}
```

Comment:
Are the id values duplicates? I'm kind of confused why the name would be used here or contain the id value since the name should be what is on the view/display. If its on the same widget since that's the prop coming from the parent maybe we can append another part to the id value in each of the inputs or pass down a little more information where this gets rendered from